src/status_im/contexts/wallet/swap/events.cljs

```diff
@@ -204,12 +204,6 @@
  (fn [{:keys [db]}]
    {:db (update-in db [:wallet :ui] dissoc :swap)}))
 
-(rf/reg-event-fx
- :wallet/on-swap-done
- (fn [_]
-   {:fx [[:dispatch [:wallet/select-account-tab :activity]]
-         [:dispatch [:wallet/clean-swap]]]}))
-
```

Comment:
@clauxx - I have kind of reverted or modified the changes introduced in #21435 by placing back the activity tab selection on successfully placing the transaction. 

Using the `:wallet/clean-swap` event on unmount Instead of the `wallet/on-swap-done` event.